RNTester/js/components/HeadingText.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
+
+import React from 'react';
+import {StyleSheet, Text} from 'react-native';
+
+type Props = $ReadOnly<{|
+  text?: ?string,
+|}>;
+
+export default function HeadingText(props: Props) {
+  return <Text style={[styles.text]}>{props.text}</Text>;
+}
+
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 20,
+    fontWeight: '500',
+    // fontFamily: 'PlayfairDisplay-Regular',
```

Comment:
Have not linked custom font's as it will cause issues with migrations. Will do it once/if a new `package.json` created after migration.